tests/args.py

```diff
@@ -164,6 +169,42 @@ def test_commands_together(self):
         self.assertEqual(args.commands, [["test", "args"], ["restart", "all"]])
 
 
+class TestProfileArguments(unittest.TestCase):
+
```

Comment:
This is a bit difficult because the paths are currently hardcoded under the `/etc/` directory.  It would require either root to run the tests (and create the files) or a script run as root to initialize the files before running.